src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java

```diff
@@ -76,6 +76,48 @@ public void testListUserStatusTransitions() throws Exception {
         assertThat(returnValue.getData().get(0).getToStatus(), is(equalTo(DE_ACTIVATED)));
     }
 
+    @Test
+    public void testListUsers_withNoParameters() throws Exception {
+        String functionality = "listUsers";
+        initMockServer(functionality);
+
+        HyperwalletList<HyperwalletUser> returnValue;
+        try {
+            returnValue = client.listUsers();
+        } catch (Exception e) {
+            mockServer.verify(parseRequest(functionality));
+            throw e;
+        }
+        assertThat(returnValue.getCount(), is(equalTo(2)));
+        assertThat(returnValue.getData().get(0).getToken(), is(equalTo("usr-00c6cfda-a43d-4b29-a20b-1abc7e800b58")));
+        assertThat(returnValue.getData().get(0).getCreatedOn(), is(equalTo(dateFormat.parse("2019-10-30T22:15:35 UTC"))));
+        assertThat(returnValue.getData().get(0).getStatus(), is(equalTo(HyperwalletUser.Status.PRE_ACTIVATED)));
+        assertThat(returnValue.getData().get(0).getVerificationStatus(), is(equalTo(HyperwalletUser.VerificationStatus.NOT_REQUIRED)));
+    }
+
+    @Test
+    public void testListUsers_withParameters() throws Exception {
+        String functionality = "listUsers";
+        initMockServer(functionality);
+
+        HyperwalletList<HyperwalletUser> returnValue;
+        try {
+            HyperwalletUsersListPaginationOptions options = new HyperwalletUsersListPaginationOptions();
+            options.clientUserId("CSLAJQt7bD");
+            //options.clientUserId("dsfdsfsdfrefdsfs");
+            returnValue = client.listUsers(options);
+        } catch (Exception e) {
+            mockServer.verify(parseRequest(functionality));
+            throw e;
+        }
+        assertThat(returnValue.getCount(), is(equalTo(2)));
+        HyperwalletUser userOne = returnValue.getData().get(0);
+        assertThat(userOne.getToken(), is(equalTo("usr-00c6cfda-a43d-4b29-a20b-1abc7e800b58")));
+        assertThat(userOne.getCreatedOn(), is(equalTo(dateFormat.parse("2019-10-30T22:15:35 UTC"))));
+        assertThat(userOne.getStatus(), is(equalTo(HyperwalletUser.Status.PRE_ACTIVATED)));
+        assertThat(userOne.getVerificationStatus(), is(equalTo(HyperwalletUser.VerificationStatus.NOT_REQUIRED)));
+    }
```

Comment:
Pls add assert statement for the 2nd index as well.

Reply:
Done